app/forms/backtaxes_form.rb

```diff
@@ -3,7 +3,15 @@ class BacktaxesForm < QuestionsForm
   validate :at_least_one_year
 
   def save
-    @intake.update(attributes_for(:intake).merge(client: Client.create!))
+    # some of the needs_help fields are not shown, based on what years the client has filed
+    # but the intake does not allow nil values
+    needs_help_params = {
+      needs_help_2018: needs_help_2018.nil? ? "unfilled" : needs_help_2018,
```

Comment:
trying to think of whether there are any implications of leaving these as unfilled further on down the line, but this actually seems to be the most respectful to the actions the client has taken in terms of altering data.